t/t2800-jobs-cmd.t

```diff
@@ -27,83 +27,132 @@ runpty="${SHARNESS_TEST_SRCDIR}/scripts/runpty.py -f asciicast"
 # state.
 #
 
+# Return the expected jobids list in a given state:
+#   "all", "run", "sched", "active", "inactive",
+#   "completed", "cancelled", "failed"
+#
+state_ids() {
+	for f in "$@"; do
+		cat ${f}.ids
+	done
+}
+
+# Return the expected count of jobs in a given state (See above for list)
+#
+state_count() {
+	state_ids "$@" | wc -l
+}
+
 wait_states() {
-        local i=0
-        while ( [ "$(flux jobs --suppress-header --filter=sched | wc -l)" != "6" ] \
-                || [ "$(flux jobs --suppress-header --filter=run | wc -l)" != "8" ] \
-                || [ "$(flux jobs --suppress-header --filter=inactive | wc -l)" != "6" ]) \
-               && [ $i -lt 50 ]
-        do
-                sleep 0.1
-                i=$((i + 1))
-        done
-        if [ "$i" -eq "50" ]
-        then
-            return 1
-        fi
-        return 0
+	sched=$(state_count sched)
+	run=$(state_count run)
+	inactive=$(state_count inactive)
+	local i=0
+	printf >&2 "Waiting for sched=$sched run=$run inactive=$inactive\n"
+	while ( [ "$(flux jobs -n --filter=sched | wc -l)" != "$sched" ] \
+	     || [ "$(flux jobs -n --filter=run | wc -l)" != "$run" ] \
+	     || [ "$(flux jobs -n --filter=inactive | wc -l)" != "$inactive" ]) \
+	&& [ $i -lt 50 ]
+	do
+	sleep 0.1
```

Comment:
doh!  I just noticed these indentations looked off

Reply:
Ok, thanks! I'll fix that up.